spec/features/merchant_items/index_spec.rb

```diff
@@ -4,37 +4,43 @@
   describe 'as a merchant' do
```

Comment:
I couldn't follow everything that was happening here, to be honest. Too many changes. :) But thanks for refactoring and incorporating enums. Looks much neater.